I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 142, in script call
    call new_round
  File "game/script.rpy", line 161, in script call
    call player_turn
  File "game/script.rpy", line 169, in script
    $ result = ui.interact() # This is a golden line in complex Ren'Py loops, you can get player input from screen with it!
  File "game/script.rpy", line 169, in <module>
    $ result = ui.interact() # This is a golden line in complex Ren'Py loops, you can get player input from screen with it!
  File "renpy/common/00action_other.rpy", line 420, in __call__
    self.callable(*self.args, **self.kwargs)
NameError: global name 'play_cards' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 142, in script call
    call new_round
  File "game/script.rpy", line 161, in script call
    call player_turn
  File "game/script.rpy", line 169, in script
    $ result = ui.interact() # This is a golden line in complex Ren'Py loops, you can get player input from screen with it!
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\ast.py", line 785, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\python.py", line 1448, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 169, in <module>
    $ result = ui.interact() # This is a golden line in complex Ren'Py loops, you can get player input from screen with it!
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\ui.py", line 277, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\core.py", line 2276, in interact
    repeat, rv = self.interact_core(preloads=preloads, **kwargs)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\core.py", line 2907, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\layout.py", line 877, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\layout.py", line 877, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\layout.py", line 877, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\screen.py", line 625, in event
    rv = self.child.event(ev, x, y, st)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\layout.py", line 877, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\layout.py", line 188, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\layout.py", line 877, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\layout.py", line 877, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\behavior.py", line 785, in event
    return handle_click(self.clicked)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\behavior.py", line 728, in handle_click
    rv = run(action)
  File "F:\Development\RenPy\renpy-6.15.7-sdk\renpy\display\behavior.py", line 290, in run
    return var(*args, **kwargs)
  File "renpy/common/00action_other.rpy", line 420, in __call__
    self.callable(*self.args, **self.kwargs)
  File "F:\Development\RenPy\SxMnGm/game\python\static.py", line 6, in set_table_status
    renpy.show(play_cards())
NameError: global name 'play_cards' is not defined

Windows-7-6.1.7601-SP1
Ren'Py 6.99.4.467
SxMnGm 0.1 alpha
